lock-api-objects/src/main/java/com/palantir/lock/v2/StartTransactionRequestV5.java

```diff
@@ -32,4 +32,13 @@
     UUID requestorId();
     OptionalLong lastKnownLockLogVersion();
     int numTransactions();
+
+    static StartTransactionRequestV5 createForRequestor(UUID requestorId, OptionalLong lockVersion, int number) {
```

Comment:
how does this tie in with conjure objects? or this "above" the rpc layer so to speak?

Reply:
Yeah some things can get cleaned up, but outside the scope of this PR

Reply:
is it within the scope of lock watches though? like if there's a chance to clear some things up, it would be ideal, before it gets too annoying to actually change etc.